### ISSUE TITLE
openCodeEditor resolves promise when clicking on Cancel button

### ISSUE THREAD

**Issue by alcazes** [adobe/reactor-turbine#156]
### Expected Behaviour
Per the documentation https://experienceleague.adobe.com/docs/experience-platform/tags/extension-dev/web/views.html?lang=en

openCodeEditor
```javascript
window.extensionBridge.openCodeEditor().then(function(code) {
  console.log(code);
});
```

Calling this method will show a modal allowing a user to edit a snippet of code. When the user has finished editing the code, the promise will be resolved with the updated code. **If the user closes the code editor without electing to save changes, the promise will never be resolved**. The options object should be structured as follows:

Per the documentation it says that if the code editor is closed without saving then the promise will not be resolved, therefore the code inside then... should not be executed

### Actual Behaviour

At present if you use the following code as per documentation
```javascript
function editCustomCodeConfig() {
    window.extensionBridge.openCodeEditor({
        code: window.customCodeJs.source,
        language: 'javascript'
    }).then(function(code) {
        console.log('Display code', code)
    });
}
```

the promise is resolved when clicking on the Cancel button of the code editor window.

The console log will log `Display code undefined`

This results in the code specified in the `then` to be executed. This results in the settings being updated even if it should not be as logic is present in the the to update the action settings.

### Reproduce Scenario (including but not limited to)

This can be reproduced both in the reactor sandbox but also in production.

Create an action with a view.
Add a button to open code editor
Add code and save >> code saved successfully
Open editor again click on cancel >> you will notice that then is executed when it should not be

**Comment by jhicken** [comment]
Well thats not good. We will put this on the back log to dig a little on.

**Comment by jeffreywalter** [comment]
@alcazes We are closing this issue on this repo and "moving" it here: https://github.com/adobe/reactor-bridge/issues/42

**Comment by brenthosie** [comment]
I am not seeing this happen. Here's what I tested.

1. Visit the lens UI
2. create the following on the window: `window.customCodeJs = { source: "the initial value" }`
3. ran the following:

`window.extensionBridge.openCodeEditor({
        code: window.customCodeJs.source,
        language: 'javascript'
    }).then(function(code) {
        console.log('Display code', code)
    });`

4. Changed the value in the opened modal then closed it.
5. Observed the Promise never resolved
6. Observed the console.log('Display code', code) never ran
7. Confirmed that window.customCodeJs was still `{ source: "the initial value"}`
